examples/documentation.html

```diff
@@ -858,6 +876,24 @@ <h3 class="option"><span class="expand all">[ <a href="javascript:$('#optionlist
             default: "count",
             refs: ["http://bl.ocks.org/kerryrodden/477c1bfb081b783f80ad"],
             examples: ["size"]
+        },
+        ylowwer: {
+            desc: "lowwer y in confidence interval.",
+            default: "function(d) { return d.ylower }",
+            examples: [
+		function(d){
+                    return d.yllower;
+                }
+	    ]
+        },
+        yupper: {
+            desc: "upper y in confidence interval.",
+            default: "function(d) { return d.yupper }",
+            examples: [
+		function(d){
+                    return d.yupper;
+                }
+	    ]

```

Comment:
You have these twice in the documentation.html...   

Also, "lower" doesn't have two w's